ie/tsn-bridge-id.go

```diff
@@ -15,9 +15,9 @@ func NewTSNBridgeID(mac net.HardwareAddr) *IE {
 		return New(TSNBridgeID, []byte{0x00})
 	}
 
-	b := make([]byte, 7)
+	b := make([]byte, 9)
 	b[0] = 0x01
-	copy(b[1:7], mac)
+	copy(b[1:9], mac)
 	return New(TSNBridgeID, b)
```

Comment:
By using the length of the input, we can still support constructing with the 6-byte MAC address. Not quite sure if it's worth, but it's basically a good idea to keep the old implementation as long as possible.

```go
l := len(mac) + 1
b := make([]byte, l)
...
```